crates/nu-command/src/conversions/into/duration.rs

```diff
@@ -19,10 +20,19 @@ impl Command for SubCommand {
             .input_output_types(vec![
                 (Type::String, Type::Duration),
                 (Type::Duration, Type::Duration),
+                // TODO: --convert option should be implemented as `format duration`
```

Comment:
It looks like some of these `into duration` changes are reverting @sholderbach's move of them to `format duration`? Is there a reason that you're doing that here instead of there?